src/styles/blocks/_stopwatch.scss

```diff
@@ -0,0 +1,55 @@
+.stopwatch {
+  position: absolute;
+  width: 500px;
+  height: 500px;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  border: 10px solid rgb(127, 255, 212);
+  background-color: #fff;
+  border-radius: 50%;
+  margin: auto;
+  box-shadow: 20px 20px 10px 15px rgba(0, 0, 255, 0.2);
+}
+
+.stopwatch__center {
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: 0;
```

Comment:
move it up by the half of `center` height (i.e. radius, radius = height / 2) – you should place right in the middle of `stopwatch` itself (hint – use variables)

Reply:
i didn't understand your this comment ! :(